versioned_docs/version-8.7/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -51,16 +51,26 @@ import Licensing from '../../../self-managed/react-components/licensing.md'
 
 ## OIDC configuration
 
-Claims are name/value pairs used to represent an individual identity. Configure your initial claim and value to match the claim used with your OIDC provider. For example, to use your Microsoft Entra unique account ID, set `IDENTITY_INITIAL_CLAIM_NAME` to `oid`, and `IDENTITY_INITIAL_CLAIM_VALUE` to the ID.
+For more information on integrating with an OpenID Connect provider, see the [OIDC documentation](/self-managed/setup/guides/connect-to-an-oidc-provider.md).
 
 :::note
+Claims are name/value pairs used to represent an individual identity. Configure your initial claim and value to match the claim used with your OIDC provider. For example, to use your Microsoft Entra unique account ID, set `IDENTITY_INITIAL_CLAIM_NAME` to `oid`, and `IDENTITY_INITIAL_CLAIM_VALUE` to the ID.
```

Comment:
Similar to my other comment, I think the product name is "Microsoft Entra ID" but I think we want to state something like "use your unique Microsoft Entra ID account ID" ... ? "use your unique Microsoft Entra ID credentials" 🤷‍♀️